khard/carddav_object.py

```diff
@@ -377,7 +384,8 @@ def _get_new_group(self, group_type=""):
             else:
                 return group_name
 
-    def _add_labelled_object(self, obj_type, user_input, name_groups=False):
+    def _add_labelled_object(self, obj_type, user_input, name_groups=False,
+                             allowed_object_type=ObjectType.string):
```

Comment:
I know this method did not have a docstring before but if you can it would be nice to add a docstring.